src/components/PageTitle.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <h1
+    class="text-[40px] font-[400] font-normal leading-[48px] color-[#34495e] not-italic">
```

Comment:
One small nit-pick here:
`font-[400] font-normal not-italic` are not required here since this is a top level component. They are not gonna be affected unless, let's say, we make the entire page italic, in which case I suppose the title should be italic too 🙂 
Also, you can combine `font-size` and `line-height` by using Tailwind typography - https://tailwindcss.com/docs/font-size 
I know that the page title is 40px in the wireframe, but I would suggest making what makes sense instead of doing pixel-perfect. If default config is close enough then it's good enough. Don't make your life more difficult 🙂